packages/components/app/styles/components/reveal.scss

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// REVEAL COMPONENT
+//
+
+:root {
+  // Spacing
+  --token-reveal-content-margin-top: 4px;
```

Comment:
Let me know if you think other values should be tokenized. (I will create JSON tokens file closer to end of development.)

Reply:
probably for such a simple component I would not bother to create a single token and just use an inlined value in the declaration.

Reply:
Ok I just went ahead and removed the CSS custom property and added the value in the styles directly. Let me know if I misunderstood what you meant though.